### PR TITLE
response body support for api security

### PR DESCRIPTION
**Pull request by christophe-papazian**
Add body response test for api security

- add new capability to tag_value end point of the weblog
- use that to test schema computation of body response for api security
- allow meta tags for api security to be gzip/b64 encoded or not (before it was mandatory, but it's not in the RFC)

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com
* [ ] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

File: utils/proxy/_deserializer.py [hunk 1]
```diff
@@ -96,7 +93,8 @@ def deserialize_dd_appsec_s_meta(key, payload):
     try:
         return json.loads(gzip.decompress(base64.b64decode(payload)).decode())
     except Exception as e:
-        raise ValueError(f"meta {key} should be b64/gziped JSON.\nPayload: {payload}") from e
+        # b64/gzip is optional
+        return payload
```
Comment (cbeauchesne) [inline utils/proxy/_deserializer.py:97]: is payload a json-string ? if yes, can you decode it ?

Comment (christophe-papazian) [reply utils/proxy/_deserializer.py:97]: nice catch, I need to fix that.

File: tests/appsec/api_security/test_schemas.py [hunk 2]
```diff
@@ -174,15 +174,26 @@ def test_request_method(self):
...
+            "/tag_value/payload_in_response_body_001/200",
+            data={"test_int": 1, "test_str": "anything", "test_bool": True, "test_float": 1.5234},
+        )
 
     def test_request_method(self):
         """can provide response body schema"""
-        pass
+        schema = get_schema(self.request, "res.body")
+        assert self.request.status_code == 200
+        print(self.request.request, self.request.status_code, self.request.headers, self.request.text)
```
[the remark this answers is not in the document]
Comment (christophe-papazian) [reply tests/appsec/api_security/test_schemas.py]: fixed

File: utils/proxy/_deserializer.py (hunk 1 above)
Comment (christophe-papazian) [reply utils/proxy/_deserializer.py]: fixed

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**